Makefile

```diff
@@ -358,12 +358,12 @@ API_REFERENCE_DIR := $(ROOT_DIR)/docs/api-reference
 
 crd-ref-docs: $(CRD_REF_DOCS) #EXHELP Generate the API Reference Documents.
 	rm -f $(API_REFERENCE_DIR)/$(OPERATOR_CONTROLLER_API_REFERENCE_FILENAME)
-	$(CRD_REF_DOCS) --source-path=$(ROOT_DIR)/api \
+	$(CRD_REF_DOCS) --source-path=$(ROOT_DIR)/api/operator-controller \
 	--config=$(API_REFERENCE_DIR)/crd-ref-docs-gen-config.yaml \
 	--renderer=markdown --output-path=$(API_REFERENCE_DIR)/$(OPERATOR_CONTROLLER_API_REFERENCE_FILENAME);
 
 	rm -f $(API_REFERENCE_DIR)/$(CATALOGD_API_REFERENCE_FILENAME)
-	$(CRD_REF_DOCS) --source-path=$(ROOT_DIR)/catalogd/api \
+	$(CRD_REF_DOCS) --source-path=$(ROOT_DIR)/api/catalogd \
```

Comment:
Can we not simplify here as well ./...

Reply:
Not if the API docs are in different locations, if we wish to consolidate the documentation, then that's a possibility.